docs/release-notes/v0.38.0.md

```diff
@@ -0,0 +1,86 @@
+# Announcing Radius v0.38.0
+
+Today we're happy to announce the release of Radius v0.38.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Secret stores in environment variables
+
+Secret stores can now be referenced in the environment variables of a container resource. Previously, the only way to provide secrets to a container resource in Radius is to mount them through a secret volume, so the experience is now much simpler for users who only require the secrets to be passed into the container via environment variables. See the updated [container resource format](https://docs.radapp.io/reference/resource-schema/core-schema/container-schema/) for more information. Many thanks to community contributor @superbeeny for adding this functionality!
+
+### Secret store reference in Dapr components
+
+Support has been added for building Dapr components with an optional reference to a Dapr secret store. This means that users may now reference non-Kubernetes secrets within the Dapr resources, closing a feature gap between Radius and the Dapr `spec.metadata` specification. See the [How-To: Reference secrets in Dapr components](https://docs.radapp.io/guides/author-apps/dapr/how-to-dapr-secrets) guide for more information. Many thanks to community contributor @SoTrx for adding this functionality!
```

Comment:
```suggestion
Support has been added to manage secrets in the Dapr components. This means that users may now reference non-Kubernetes secrets within the Dapr resources, closing a feature gap between Radius and the Dapr. See the [How-To: Reference secrets in Dapr components](https://docs.radapp.io/guides/author-apps/dapr/how-to-dapr-secrets) guide for more information. Many thanks to community contributor @SoTrx for adding this functionality!
```

Reply:
@willtsai - Rephrased a bit and removed `spec.metadata` to make it clear. Let me know if the edit makes sense.